packages/api/src/hooks/useAccountTypes.ts

```diff
@@ -0,0 +1,32 @@
+import { useMemo } from 'react';
+import useFetch from '../useFetch';
+
+/**
+ * A custom hook to get available account types for a specific landing company
+ *
+ *  @param landing_company {string} - The landing company shortcode
+ */
+const useAccountTypes = (landing_company?: string) => {
+    const { data } = useFetch('get_account_types', {
+        payload: { company: landing_company },
+        options: { enabled: Boolean(landing_company) },
```

Comment:
@farhan-nurzi-deriv `landing_company` parameter is optional 🤔 Should we still disable the call if not provided? 🤔

Reply:
If not provided it will fallback to active landing company 👀

Reply:
Yeah we shouldnt disable it actually 👀